scripts/playout_controls.sh

```diff
@@ -365,6 +367,14 @@ case $COMMAND in
     getidletime)
         echo $IDLETIME
         ;;
+	enablewifi)
+		rfkill unblock wifi
+		;;
+	disablewifi
```

Comment:
Hi @mangiari 
I get it now. And it makes sense. And I believe that electro smog might eventually turn out to be what radio activity was only decades ago: an invisible and therefore underestimated hazzard.
However, please check the code an then create a pull request again. There is an `)` missing in this line.
Thanks, micz